tests/cli/test_vehicle.py

```diff
@@ -273,6 +282,27 @@ def test_vehicle_status_no_prompt(
 
     assert EXPECTED_STATUS["zoe_40.1.json"] == result.output
 
+def test_vehicle_status_json(
```

Comment:
Missing carriage return here.
```suggestion

def test_vehicle_status_json(
```